guidelines/monai-application-package.md

```diff
@@ -1,498 +1,650 @@
 # MONAI Application Package
 
-## Description
+<!-- @import "[TOC]" {cmd="toc" depthFrom=2 depthTo=3 orderedList=false} -->
 
-This is a proposal for the MONAI Deploy Working Group.
+<!-- code_chunk_output -->
 
+- [Description](#description)
 - [Overview](#overview)
   - [Goal](#goal)
   - [Assumptions](#assumptions)
 - [Requirements](#requirements)
-  - [Contains an Application](#contains-an-application)
   - [Single Artifact](#single-artifact)
   - [Self-Describing](#self-describing)
+  - [Runtime Characteristics of the MAP](#runtime-characteristics-of-the-map)
   - [IO Specification](#io-specification)
   - [Local Execution](#local-execution)
-  - [Containerization](#containerization)
   - [Compatible with Kubernetes](#compatible-with-kubernetes)
   - [OCI Compliance](#oci-compliance)
-  - [Facilitate GPU Acceleration](#facilitate-gpu-acceleration)
-- [Architecture & Design](#architecture--design)
-- [Description](#description)
+  - [Hosting Environment](#hosting-environment)
+- [Description](#description-1)
 - [Application](#application)
 - [Manifests](#manifests)
   - [Application Manifest](#application-manifest)
   - [Package Manifest](#package-manifest)
-- [Executor](#executor)
-  - [Initial Conditions / Environment Variables](#initial-conditions)
-  - [Manifest Export](#manifest-export)
+- [Supplemental Application Files](#supplemental-application-files)
+  - [Container Behavior and Interaction](#container-behavior-and-interaction)
   - [Table of Important Paths](#table-of-important-paths)
-- [Package Layout Diagram](#layout-diagram)
+- [Package Layout Diagram](#package-layout-diagram)
 
+<!-- /code_chunk_output -->
 
 ## Overview
 
-This proposal documents the specification of the initial version of the MONAI Application Package (MAP).
-
+This document includes the specification of the MONAI Application Package (MAP).
```

Comment:
```suggestion
This document includes the specification of the MONAI Application Package (MAP). A MAP is a containerized application or service which is self-descriptive as defined by this document.
```

I would add just a little more explanation that is generic enough to encompas what MAPs can be.

Reply:
Updated as suggested